client/src/components/details/image.jsx

```diff
@@ -2,12 +2,25 @@ import React from 'react';
 import Stars from '../../assets/Stars.jsx';
 import StyleLib from '../../assets/Stylesheet.jsx';
 import DetailsLib from './assets/Stylesheet.jsx';
+import Carousel from 'react-responsive-carousel';
 
-const Image = () => {
+const Image = ({styleList, setOpenModal}) => {
+  // TODO: return, change default product to be currently selected style instead
+  var defaultStyle = styleList[0];
+  for (var i = 0; i < styleList.length; i++) {
+    if (styleList[i]['default?']) {
+      defaultStyle = styleList[i];
+    }
```

Comment:
you could also write this as:
`const defaultStyle = styleList.filter(style => style['default?'])[0]`
which is a little cleaner and takes up less space :)